pysac/mhs_atmosphere/parameters/model_pars.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Dec 11 17:45:48 2014
+
+@author: sm1fg
+"""
+
+import numpy as np
+import sys
+
+def get_model(model, 
```

Comment:
Ah balls, you de-dimensionalise the values here, so it will have to be in a function.